tests/wdio/raw-spec.js

```diff
@@ -0,0 +1,11 @@
+Terra.describeViewports('raw route', ['tiny', 'huge'], () => {
+  it('checks accessibility on the raw route', () => {
+    browser.url('/raw/secondary-nav-test/terra-dev-site/themed');
+    Terra.validates.element({ selector: '#root' });
```

Comment:
I noticed we are adding a custom selector in each of these tests. Did the global not meet our needs? Should we update the global selector for dev site or could we update the pages to have height 100% to match the global selector most people will be using by default?

Reply:
Terra-dev-site's tests are different in that in most tests we're testing the frame and not the content. Any consumer of dev site would use the normal selector.